server/migrations/20191010101148-0.0.9.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-param-reassign */
+const { runAsyncTasks } = require('../common/utils');
+const { EmailNotificationsFrequency } = require('../common/variables.js');
+
+const up = async db => {
+  const addEmailNotificationSettings = async () =>
+    db.collection('users').updateMany(
+      {
+        emailNotificationsFrequency: { $exists: false },
+        lastEmailNotificationSentAt: { $exists: false }
+      },
+      {
+        $set: {
+          emailNotificationsFrequency: EmailNotificationsFrequency.NEVER,
+          lastEmailNotificationSentAt: ''
```

Comment:
type of lastEmailNotificationSentAt is Date. When there is no value we should save an empty string or null? OR maybe it is unnecessary to set it now maybe at the first sending we should save it to user doc?